### DISCUSSION TITLE
[QUESTION] How can one contribute to `studio` tool?

### DISCUSSION THREAD

**Discussion by anasouardini** [prisma/prisma#23420] [Q&A]
### Question

Sorry if this is a dumb question, but I can only find the `npm` package for the studio tool and the CLI command/script for it in the source code.

**Comment by ludralph** [comment]
Hi @anasouardini 👋 

Thank you for raising this question.

The Prisma Studio repository is not Open Source. If you want us to work on a specific feature for Studio, you could report it by creating an issue in this repository.

- https://github.com/prisma/studio

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
